From 51c4206c4a9b7fae2431f27a98de6bc8ba32922f Mon Sep 17 00:00:00 2001
From: GUO Zihua <guozihua@huawei.com>
Date: Wed, 2 Aug 2023 10:14:36 +0800
Subject: [PATCH 063/280] x86/sev: Make boot_ghcb_page[] static

boot_ghcb_page is not used by any other file, so make it static.

This also resolves sparse warning:

  arch/x86/boot/compressed/sev.c:28:13: warning: symbol 'boot_ghcb_page' was not declared. Should it be static?

Signed-off-by: GUO Zihua <guozihua@huawei.com>
Signed-off-by: Ingo Molnar <mingo@kernel.org>
Cc: linux-kernel@vger.kernel.org
---
 arch/x86/boot/compressed/sev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/boot/compressed/sev.c b/arch/x86/boot/compressed/sev.c
index 80d76aea1f7b..ab3c6927d5de 100644
--- a/arch/x86/boot/compressed/sev.c
+++ b/arch/x86/boot/compressed/sev.c
@@ -25,7 +25,7 @@
 #include "error.h"
 #include "../msr.h"
 
-struct ghcb boot_ghcb_page __aligned(PAGE_SIZE);
+static struct ghcb boot_ghcb_page __aligned(PAGE_SIZE);
 struct ghcb *boot_ghcb;
 
 /*
-- 
2.43.0

